src/waypoint.jsx

```diff
@@ -169,8 +167,8 @@ export default class Waypoint extends React.Component {
     if (node.offsetParent === this.scrollableAncestor || !node.offsetParent) {
       return node.offsetTop;
     } else {
-      return node.offsetTop +
-        this._distanceToTopOfScrollableAncestor(node.offsetParent);
+      let nextOffset = this._distanceToTopOfScrollableAncestor(node.offsetParent);
```

Comment:
The build is failing because you don't need `let` here. We lint the code (eslint) as part of testing, and this error comes up:
```
 170:11  error    `nextOffset` is never modified, use `const` instead  prefer-const
```